### PR TITLE
[15.0][MIG] base_rest_demo: Migration to 15.0

### PR DESCRIPTION
**Pull request by yankinmax**
depends on:

* [x] #222
* [x] #225
* [x] #230

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+odoo-addon-base_rest @ git+https://github.com/OCA/rest-framework@refs/pull/225/head#subdirectory=setup/base_rest
```
Comment (simahawk) [inline test-requirements.txt]: pls ALWAYS use  a separated commit for test-requirements w/ proper msg like "Add TMP test dependency"

Comment (yankinmax) [reply test-requirements.txt]: I forgot second time)))) Sure.

**Comment by yankinmax** [comment]
@lmignon Can you pls take a look here?

**Review by simahawk — COMMENTED** [review]
LG

**Comment by yankinmax** [comment]
@simahawk @lmignon can you pls take a look?
This module stays mostly unchanged.
But what is strange for me that runboat/build here continue staying in pending for two days
build
.

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@yankinmax not blocking ATM. Worth to notify @sbidoul tho. See https://runboat.odoo-community.org/webui/build.html?name=b02276cb9-bec3-4b6a-ad7f-e9434a2b0b8f

**Review by marcos-mendez — APPROVED** [review]
LGTM testes on local LXC

**Pull request merged**